### PR TITLE
Add support for boolean values in open office files that were generat…

### LINKED ISSUE #579: Add support for boolean values in open office files exported via Google Sheets
### Issue
Generated Open Office .ods files from a Google Sheet does not properly resolve boolean cell value types when using roo.
The generated file entry looks as follows:
```
<table:table-cell table:style-name="ce2" table:formula="of:=TRUE()" office:value-type="float" office:value="1" calcext:value-type="float">
```

It would be great if roo would support such an edge case and properly export these values as boolean strings ('true' | 'false') and not as integers.

### System configuration
**Roo version**: 2.9.0

**Ruby version**: 2.7.0, 3.1.2

### Example file

[boolean-from-google-sheets.ods](https://github.com/roo-rb/roo/files/10493143/boolean-from-google-sheets.ods)

### PR DESCRIPTION
**Pull request by patrickkulling**
###Description

Resolves #579

@kakubin here is my first fix. Please have a look - happy for feedback.

### CODE REVIEW

**Comment by patrickkulling** [comment]
@kakubin can you prioritize this PR as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ##  Unreleased
 - Fix gsub! usage for open office documents on a frozen string [580](https://github.com/roo-rb/roo/pull/580)
+- Add support for boolean values in open office files that were generated via Google Sheets [579](https://github.com/roo-rb/roo/pull/579)
```
Comment (kakubin) [inline CHANGELOG.md:3]: This new line should be end with link https://github.com/roo-rb/roo/pull/580

**Review by kakubin — APPROVED** [review]
Everything is fine except CHANGELOG.md

File: CHANGELOG.md (hunk 1 above)
Comment (patrickkulling) [reply CHANGELOG.md:3]: Let me fix this up. Must have been mixed up in the merge

Comment (patrickkulling) [reply CHANGELOG.md]: @kakubin Have another look now - the other line in the changelog was wrong as well. Both were using the issues numbers and not the PR numbers.

**Review by kakubin — APPROVED** [review]

**Pull request merged**